Ecommerce/index.css

```diff
@@ -1 +1,94 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Shop</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+            color: #333;
+        }
+        header {
+            background-color: #333;
+            color: #fff;
+            padding: 10px 20px;
+            text-align: center;
+            position: relative; /* Added position relative */
+        }
+        .brand-image {
+            position: absolute; /* Added position absolute */
+            top: 50%; /* Positioned vertically in the middle */
+            right: 20px; /* Positioned on the right side */
+            transform: translateY(-50%); /* Adjusting to vertically center */
+            max-width: 100px; /* Adjust size as needed */
+            height: auto;
+        }
+        .container {
+            max-width: 500px;
+            margin: 20px auto;
+            padding: 0 20px;
+        }
+        img {
+            max-width: 100%;
+            height: auto;
+            display: block;
+            margin: 20px auto;
+        }
+        form {
+            margin-top: 20px;
+            background-image: url('https://img.freepik.com/premium-vector/seamless-pattern-sketches-shopping-bags_615502-3186.jpg?w=740'); /* Add your background image */
+            padding: 40px; /* Add padding to see the background */
+            border-radius: 10px; /* Add border radius for better aesthetics */
+        }
+        input[type="text"], input[type="email"], textarea {
+            width: 100%;
+            padding: 10px;
+            margin-bottom: 10px;
+            border: 1px solid #ccc;
+            border-radius: 5px;
+            box-sizing: border-box;
+        }
+        input[type="submit"] {

```

Comment:
Consider adding a comment above 'background-color' property in input[type="submit"] to describe its significance for better code documentation.